From 654dd1b4c5e82c44fbaf8d04eac083e006a3fb62 Mon Sep 17 00:00:00 2001
From: rodriguezst <2828844+rodriguezst@users.noreply.github.com>
Date: Fri, 31 Jan 2025 20:35:40 +0100
Subject: [PATCH] Set UFS version to 3 instead of 2

---
 .../Library/AcpiPlatformUpdateLib/AcpiPlatformUpdateLib.c     | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Silicon/QC/Sm8150/QcomPkg/Library/AcpiPlatformUpdateLib/AcpiPlatformUpdateLib.c b/Silicon/QC/Sm8150/QcomPkg/Library/AcpiPlatformUpdateLib/AcpiPlatformUpdateLib.c
index de12fd148..1eb800858 100644
--- a/Silicon/QC/Sm8150/QcomPkg/Library/AcpiPlatformUpdateLib/AcpiPlatformUpdateLib.c
+++ b/Silicon/QC/Sm8150/QcomPkg/Library/AcpiPlatformUpdateLib/AcpiPlatformUpdateLib.c
@@ -89,7 +89,7 @@ PlatformUpdateAcpiTables(VOID)
   UINT16                              SDFE  = 0;
   UINT16                              SIDM  = 0;
   UINT32                              SUFS  = 0xFFFFFFFF;
-  UINT32                              PUS3  = 0x0;
+  UINT32                              PUS3  = 0x1;
   UINT32                              SUS3  = 0xFFFFFFFF;
   UINT32                             *pSIDT = (UINT32 *)0x784130;
   UINT32                              SIDT  = (*pSIDT & 0xFF00000) >> 20;
@@ -228,4 +228,4 @@ PlatformUpdateAcpiTables(VOID)
 
   // Increase CPU speed for HLOS
   SetupAPSSCpuPerformanceLevels();
-}
\ No newline at end of file
+}
